packages/web-app-files/src/mixins/fileActions.js

```diff
@@ -241,6 +241,7 @@ export default {
       const routeData = this.$router.resolve({
         name: 'external-apps',
         params: {
+          contextRouteName: this.$route.name,
```

Comment:
I know I'm late but this would have been a chance to introduce a common helper for params to external and the other apps.

Reply:
Would you like to do that in a separate PR? 😇 and/or create an issue? 😊